src/Symfony/Component/Workflow/Exception/TransitionException.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Workflow\Exception;
+
+/**
+ * @author Andrew Tch <andrew.tchircoff@gmail.com>
+ */
+class TransitionException extends LogicException implements ExceptionInterface
```

Comment:
~implements ExceptionInterface~ (since already implemented by `LogicException` now :)

Reply:
Done. Sorry, my coding style is chaotic somehow )